api/worker.go

```diff
@@ -115,22 +115,24 @@ type (
 
 	// RHPRenewRequest is the request type for the /rhp/renew endpoint.
 	RHPRenewRequest struct {
-		ContractID    types.FileContractID `json:"contractID"`
-		EndHeight     uint64               `json:"endHeight"`
-		HostAddress   types.Address        `json:"hostAddress"`
-		HostKey       types.PublicKey      `json:"hostKey"`
-		SiamuxAddr    string               `json:"siamuxAddr"`
-		NewCollateral types.Currency       `json:"newCollateral"`
-		RenterAddress types.Address        `json:"renterAddress"`
-		RenterFunds   types.Currency       `json:"renterFunds"`
-		WindowSize    uint64               `json:"windowSize"`
+		ContractID         types.FileContractID `json:"contractID"`
+		EndHeight          uint64               `json:"endHeight"`
+		ExpectedNewStorage uint64               `json:"expectedNewStorage"`
+		HostAddress        types.Address        `json:"hostAddress"`
+		HostKey            types.PublicKey      `json:"hostKey"`
+		MinNewCollateral   types.Currency       `json:"minNewCollateral"`
+		SiamuxAddr         string               `json:"siamuxAddr"`
+		RenterAddress      types.Address        `json:"renterAddress"`
+		RenterFunds        types.Currency       `json:"renterFunds"`
+		WindowSize         uint64               `json:"windowSize"`
 	}
 
 	// RHPRenewResponse is the response type for the /rhp/renew endpoint.
 	RHPRenewResponse struct {
 		Error          string                 `json:"error"`
 		ContractID     types.FileContractID   `json:"contractID"`
 		Contract       rhpv2.ContractRevision `json:"contract"`
+		ContractPrice  types.Currency         `json:"contractPrice"`
```

Comment:
Does it make sense to drop `ContractID` from `RHPRenewResponse` ? It's duplicated info.
Does it make sense to return the `renewCollateral` directly instead of the `ContractPrice`?

Reply:
The `ContractPrice` is the thing we actually need for the next API call. So I'm returning it instead of the thing we only need for logging. At the end of the day it doesn't really matter cause the endpoint will be removed with rhpv4 anyway once we remove anything related to contracts to the bus.